contents/articles/2016-11_focal-musical-expression/index.md

```diff
@@ -0,0 +1,37 @@
+--- 
+title: An Eye-Tracking Musical Expression Controller
+date: Fri Nov 11 2016 14:43:37 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Focal is an experimental concept that uses AR headset with eye tracking as a digital musical controller..."
+featured_img: "../../../../media/images/blog/focal-systemb.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/focal-music.jpg"
+---
+
+Focal is an exciting experimental concept that uses AR headset with eye tracking as a digital musical controller. Enabling the musician to control many virtual electronic audio effects and synthesis parameters. The performer can control the virtual dials and switches by glancing at the effects they want to activate and using a single expression pedal to adjust those effects, while maintaining posture and balance for minimal disturbance and interaction during performance.
+
+<div class="Grid Grid--center Grid--justifyCenter">
+	<img class=".Feature-image--capturePlayerIcons
+	" src="../../../../media/images/blog/focal-system.jpg" class='Feature-image u-padBottom--1' alt="Focal System">
+</div>
+Image Source: [Focal](http://stewartgreenhill.com/articles/focal/)
+<br>
+<br>
+
+The Focal system consists of four main parts:
+- Epson Moverio AR headset
+- Pupil eye-tracker add-on
+- 2 Pedals
+- Focal application
+
+The headset combines a Pupil Epson Moverio monocular mount add-on with the Epson Moverio BT-200 AR glasses. To track the gaze of the musician and display the Focal application. And many of the electronic effects in a multi-pedal setup otherwise used physically in a performance is now displayed on the HUD where musicians can interact with interface.
```

Comment:
Edit: 
Focal uses the Epson Moverio BT-200 with a Pupil eye tracking [add-on](link to store) and the developers/musicians have provided excellent documentation of their process! Check out their writeup [here](link).